catroid/src/org/catrobat/catroid/formulaeditor/FormulaElement.java

```diff
@@ -739,6 +745,18 @@ private Object interpretObjectSensor(Sensors sensor, Sprite sprite) {
 			case OBJECT_Y_VELOCITY:
 				returnValue = (double) sprite.look.getYVelocityInUserInterfaceDimensionUnit();
 				break;
+			case OBJECT_LOOK_NUMBER:
+				returnValue = 1.0d + ((lookData != null) ? lookDataList.indexOf(lookData) : 0);
+				break;
+			case OBJECT_LOOK_NAME:
+				returnValue = (lookData != null) ? lookData.getLookName() : null;
+				break;
+			case OBJECT_BACKGROUND_NUMBER:
+				returnValue = 1.0d + ((lookData != null) ? lookDataList.indexOf(lookData) : 0);
+				break;
+			case OBJECT_BACKGROUND_NAME:
+				returnValue = (lookData != null) ? lookData.getLookName() : null;
```

Comment:
Will there be a problem when returning `null`?
e.g. concating two strings, `join(Hello, !background.name which will return null!)` = ?